Azure: Fixed the captured VHD name

iamshital opened the pull request:
Added RGIdentifier to captured VHD Name to isolate multiple captured VHDs in same build.

ader1990 commented:
Partially addresses issue:
https://github.com/LIS/LISAv2/issues/441

ader1990 commented:
This PR refactors the code so that all the methods go into their corresponding relevant file names.
Remove Libraries/Framework-Hyperv.psm1 (move the code to other files, more info below)
Remove Libraries/Framework-Azure.psm1 (move the code to other files, more info below)

Create a Libraries/LISAv2-Framework.psm1 file.
All the code from Libraries/Framework-Azure.psm1 should go into LISAV2-Framework.psm1

In Libraries/LISAv2-Framework.psm1 we should add the following methods:
Get-SecretParams (from Libraries/Framework-Hyperv.psm1)
Parse-TestParameters (from Libraries/Framework-Hyperv.psm1)
Run-SetupScript (from Libraries/Framework-Hyperv.psm1)
Create-ConstantsFile (from Libraries/Framework-Hyperv.psm1)
Run-TestScript (from Libraries/Framework-Hyperv.psm1)
Collect-TestLogs (from Libraries/Framework-Hyperv.psm1)
Run-Test (from Libraries/Framework-Hyperv.psm1)
The following methods from Libraries/Framework-Hyperv.psm1:

Enable-RootUser should go to CommonFunctions.psm1
Create-HyperVCheckpoint should go to HyperV.psm1
Create-HyperVCheckpoint should go to HyperV.psm1
Check-IP should go to HyperV.psm1

juhlee-microsoft commented:
@ader1990 This PR shows files renaming and parts are moving to another file. What is your purpose for this change?

iamshital committed:
Fixed captured VHD name

Added RGIdentifier to captured VHD Name to isolate multiple captured VHDs in same build.
